test/index.js

```diff
@@ -403,44 +406,6 @@ describe('DDWAF', () => {
     }
   })
 
-  it('should parse values correctly', () => {
```

Comment:
Why delete this test ? you could just update it so it matches the new behavior no ?

Reply:
Updated